scs2-session-visualizer-jfx/src/main/resources/fxml/menu/YoChartMenu.fxml

```diff
@@ -16,6 +16,16 @@
             </FontIcon>
          </graphic>
       </MenuItem>
+      <MenuItem onAction="#openPieChartWindow" text="Pie Chart Window">
```

Comment:
FXML files are required to load gui elements in JavaFX, had to create a few for the pie chart